source/postprocess/basic_statistics.cc

```diff
@@ -58,6 +61,14 @@ namespace aspect
               std::vector<double> composition_values(this->n_compositional_fields(),0.0);
 
               Point<dim> representative_point = this->get_geometry_model().representative_point(0);
+
+              in.position[0] = representative_point;
+              in.temperature[0] = dT;
+              in.pressure[0] = dT; //TODO: dT for the pressure is wrong
```

Comment:
I don't know. I just emulated the original behavior of this plugin. I have the impression this plugin isn't very good, but that's a separate issue.